src/web/app/pages-help/instructor-help-page/instructor-help-getting-started/instructor-help-getting-started.component.html

```diff
@@ -0,0 +1,213 @@
+<h1 id="Top" class="color-orange">Getting Started</h1>
+<div id="contentHolder">
+  <p>
+    Welcome to TEAMMATES!
+  </p>
+  <p>
+    To get started using TEAMMATES, follow the following steps, or watch our <a href="https://www.youtube.com/embed/mDtfmNmRwBM?autoplay=1&rel=0" target="_blank"><i class="fas fa-film"></i> Video Tour</a>.<br>
+    For more help, browse the answers to some <a routerLink="/web/front/help/instructor">frequently asked questions</a>.
+  </p>
+  <ol>
+    <li><a href="#course-setup">Set up a course</a></li>
+    <li><a href="#session-setup">Create a session</a></li>
+    <li><a href="#session-invites">Wait for your session to open</a></li>
+    <li><a href="#session-results">View and publish session results</a></li>
+    <li><a href="#other-actions">Learn about other actions you can perform</a></li>
+    <li><a href="#contact-us">Contact us</a></li>
+  </ol>
+  <div class="separate-content-holder">
+    <hr>
+  </div>
+  <h2 id="course-setup">1. Set up a course</h2>
```

Comment:
not sure if the id is still necessary since previously in `jsp`, it allows the page to be directed to it. Guess you will need a different implementation?

Reply:
Found a way to do the jumping around using HTML id and `document.getElementById()`. The `jumpTo()` method is implemented in the ts file of the component

PS: In the html file for the list of links at the beginning, I used `<a (click)="!!jumpTo(...)" href>` to maintain the link styling (blue color, underline and cursor) and for accessibility reasons.
Taken from [here](https://github.com/ng-bootstrap/ng-bootstrap/issues/142#issuecomment-231587330)